Resources/doc/fosuser-bundle.md

```diff
@@ -1,3 +1,109 @@
 # FOSUser Bundle integration
 
 This bundle is shipped with a bridge for the [FOSUserBundle](https://github.com/FriendsOfSymfony/FOSUserBundle). If the FOSUserBundle is enabled, this bridges registers to the persist, update and delete events to pass user objects to the UserManager, before redispatching the event. 
+
+### Example to create User with serialization groups :
```

Comment:
Change it by the following according to our doc style:

`# Creating a `User` entity with serialization groups`